### PR TITLE
Run CI tests using Gihub actions

### PR DESCRIPTION
**Pull request by cedric-anne**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -

Run full test suite in Github Action on push event. For now, it does almost same tests as what is done on CircleCI, but on a more compete matrix. PR for 9.5 branch will show some error we did not detect with CircleCI.

### CODE REVIEW

**Comment by trasher** [comment]
Sounds ok to me :)
maybe we should only keep lint and php7.3/mysql8?

**Comment by orthagh** [comment]
> lint and php7.3/mysql8

agree with that. 

Waiting for new features (specialy cache) and end of beta of actions.
I would like to view the time with alpine image also.

**Comment by cedric-anne** [comment]
I remove the exclusion list and limited tests to PHP 7.3 with MySQL 8 and MariaDB 10.4. The idea is to have MariaDB as reference environment.

I also added LDAP tests.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+      - name: "Unit tests"
+        run: |
+          docker exec ${{ job.services.app.id }} vendor/bin/atoum -p 'php -d memory_limit=512M' --debug --force-terminal --use-dot-report --configurations tests/telemetry.php --bootstrap-file tests/bootstrap.php --no-code-coverage -d tests/units
+      - name: "Functionnal tests"
+        run: |
+          docker exec ${{ job.services.app.id }} vendor/bin/atoum -p 'php -d memory_limit=512M' --debug --force-terminal --use-dot-report --configurations tests/telemetry.php --bootstrap-file tests/bootstrap.php --no-code-coverage --max-children-number 1 -d tests/functionnal
+      - name: "LDAP tests"
+        run: |
+          for f in `ls tests/LDAP/ldif/*.ldif`; do cat $f | docker exec --interactive ${{ job.services.openldap.id }} ldapadd -x -H ldap://127.0.0.1:3890/ -D "cn=Manager,dc=glpi,dc=org" -w insecure ; done
+          docker exec ${{ job.services.db.id }} mysql --user=root --execute="UPDATE \`glpi\`.\`glpi_authldaps\` SET \`host\` = 'openldap';"
```
Comment (cedric-anne) [inline .github/workflows/ci.yml]: On Travis CI, LDAP server is reacheable on `127.0.0.1`, but here, we have to access it using its network alias `openldap`.
This SQL request may be removed if we drop Travic CI tests.

**Pull request merged**